<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pause">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deal_back">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deal_next">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>100</width>
     <height>150</height>
    </rect>
   </property>
   <widget class="QPushButton" name="mute">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>70</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>静音</string>
    </property>
   </widget>
   <widget class="QSlider" name="volume">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>10</y>
      <width>30</width>
      <height>100</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="voice">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/volume.bmp</normaloff>:/volume.bmp</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QSlider" name="music_long">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>640</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>620</width>
     <height>345</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget
{  
   background-color:rgb(242,241,240);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_music">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>添加音乐</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>430</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nowtime">
   <property name="geometry">
    <rect>
     <x>533</x>
     <y>430</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="words">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>435</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>词</string>
   </property>
  </widget>
  <widget class="label_roll" name="name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>145</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="lyric" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>398</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">widget
{
	border-width: 10px;
	border-style: solid;
	border-color: purple;
	padding: 20px;
	margin: 20px;

}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Purisa</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>歌手:</string>
    </property>
   </widget>
   <widget class="QLabel" name="x2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>75</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>专辑:</string>
    </property>
   </widget>
   <widget class="QLabel" name="singer">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>310</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>singer</string>
    </property>
   </widget>
   <widget class="QLabel" name="album">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>75</y>
      <width>310</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>album</string>
    </property>
   </widget>
   <widget class="QLabel" name="front_front">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>120</y>
      <width>310</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="front">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>165</y>
      <width>310</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="center">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>215</y>
      <width>310</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color:rgb(36, 145, 246);
}</string>
    </property>
    <property name="text">
     <string>未找到歌词！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="next">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>280</y>
      <width>310</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="next_next">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>325</y>
      <width>310</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>x1</zorder>
   <zorder>x2</zorder>
   <zorder>singer</zorder>
   <zorder>album</zorder>
   <zorder>front_front</zorder>
   <zorder>front</zorder>
   <zorder>center</zorder>
   <zorder>next</zorder>
   <zorder>next_next</zorder>
   <zorder>label</zorder>
  </widget>
  <zorder>lyric</zorder>
  <zorder>listWidget</zorder>
  <zorder>pause</zorder>
  <zorder>deal_back</zorder>
  <zorder>deal_next</zorder>
  <zorder>voice</zorder>
  <zorder>music_long</zorder>
  <zorder>add_music</zorder>
  <zorder>widget</zorder>
  <zorder>time</zorder>
  <zorder>nowtime</zorder>
  <zorder>words</zorder>
  <zorder>name</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>label_roll</class>
   <extends>QLabel</extends>
   <header location="global">label_roll.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
